src/app.tsx

```diff
@@ -9,7 +10,7 @@ const App: FunctionComponent = () => {
   const [metamaskConnected, setMetamaskConnected] = useState(false);
   useEffect(() => {
     checkMetaMaskAccount();
-  });
+  }, []);
```

Comment:
this will check only once, are you sure it's what you want to do ?

Reply:
yup, i just need it to check when user load the webpage